opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -909,7 +909,8 @@ const rebuildAndMergeInputFromExistingData = (rawInput, instance) => {
         finalVal = value;
       }
     }
-    if (compareUnsorted(finalVal ?? [], currentValues)) {
+    // TODO: solve case where ordering is important and we should use regular 'compare'
+    if (key !== 'overview_layout_customization' && compareUnsorted(finalVal ?? [], currentValues)) {
```

Comment:
I'm facing an issue with this check : I'm sending an update on an array field that is a simple reordering of the array.
The check detects no change since we use `compareUnsorted`.

This is a temporar fixy, we will find a better solution and implement it rapidly.

We can:
* remove this optimization
* remove this optimization for a static set of keys, like `overview_layout_customization`
* add an attribute in the schema for "multiple" attribute, like "preserveOrder" to use `compare` instead of `compareUnsorted` here
* ... any better idea?

Reply:
ping @richard-julien WDYT ?